CONTRIBUTING.md

```diff
@@ -1,26 +1,33 @@
 # How To Contribute
 
+This repo is divided into multiple packages using Yarn workspaces:
+
+- `addon` is the actual ember-modifier addon
+- `test-app` is its test suite
+
 ## Installation
 
-* `git clone <repository-url>`
+* `git clone https://github.com/ember-modifier/ember-modifier.git`
 * `cd ember-modifier`
 * `yarn install`
 
 ## Linting
 
-* `yarn lint:hbs`
-* `yarn lint:js`
-* `yarn lint:js --fix`
+Inside any of the packages you can run:
+
+* `yarn lint`
+* `yarn lint:fix`
 
 ## Running tests
 
-* `ember test` – Runs the test suite on the current Ember version
-* `ember test --server` – Runs the test suite in "watch mode"
-* `ember try:each` – Runs the test suite against multiple Ember versions
+* `cd addon && pnpm run start` – Builds the addon in "watch mode" so changes picked up by test app.
+* `cd test-app && ember test` – Runs the test suite on the current Ember version
+* `cd test-app && ember test --server` – Runs the test suite in "watch mode"
+* `cd test-app && ember try:each` – Runs the test suite against multiple Ember versions
```

Comment:
@chriskrycho correct, you need to do both and that's one of the biggest difference when it comes to v1 vs v2 addon development ergonomics.

Actually I forgot to mention one more thing:
in order for the `ember-auto-import` to automatically pick up changes made in the addon (whilest `yarn start` is running), one actually need to do `EMBROIDER_REBUILD_ADDONS=ember-modifier ember s`.
Without `EMBROIDER_REBUILD_ADDONS`, changes won't be picked up by test app.

I'm thinking if it would be more ergonomic to do `process.env.EMBROIDER_REBUILD_ADDONS = "ember-modifier"` in the `ember-cli-build.js` instead of being mentioned in `CONTRIBUTING.md` as it probably would be overlooked.

Reply:
I like putting it in the test-app `ember-cli-build.js` for now. This is a thing I expect we'll figure out as part of the app authoring story in *general* (because we need to!).

Reply:
done!